src/core/qgsvectorlayer.h

```diff
@@ -803,6 +803,12 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
     bool writeStyle( QDomNode& node, QDomDocument& doc, QString& errorMessage ) const override;
 
     bool writeSld( QDomNode& node, QDomDocument& doc, QString& errorMessage ) const;
+
+    /**
+     * @note not available in Python bindings
+     */
+    bool writeSld( QDomNode& node, QDomDocument& doc, QString& errorMessage, QgsStringMap props = QgsStringMap() ) const;
```

Comment:
this looks odd - you've noted here that this version isn't available in the python bindings, but then you've modified the sip bindings to include this. What was the intention here?